### PR TITLE
New Workflow `Rollback`

### PR DESCRIPTION
**Pull request by Benbentwo**
## what
* Rolls back environments defined in `.github/workflows/rollback.yaml` to a supplied tag.

## why
* Environments might need to be rolled back if a critical bug is found

### CODE REVIEW

File: .github/workflows/ecspresso-rollback.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      - name: Sanitize Organization
+        id: lowercase_org
+        uses: cloudposse/github-action-string-transformer@main
+        with:
+          operation: 'lowercase'
+          input-string: ${{ inputs.organization }}
+
+      - name: Sanitize Repository
+        id: lowercase_repo
+        uses: cloudposse/github-action-string-transformer@main
```
Comment (milldr) [inline .github/workflows/ecspresso-rollback.yml:78]: should this point to a version?

File: .github/workflows/rollback.yaml
Comment (milldr) [inline .github/workflows/rollback.yaml:1]: empty file?

File: .github/workflows/ecspresso-rollback.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        with:
+          operation: 'lowercase'
+          input-string: ${{ inputs.repository }}
+
+      - uses: cloudposse/github-action-secret-outputs@0.1.1
```
Comment (milldr) [inline .github/workflows/ecspresso-rollback.yml:83]: please add step "name"

**Review by milldr — CHANGES_REQUESTED** [review]
couple of minor comments plus the `Feature Branch / do / cd / cleanup (qa1) (pull_request)` check is failing

**Comment by milldr** [comment]
Is this the same as a `hotfix` workflow? Wondering if we should rename `rollback` to `hotfix` for consistency

File: .github/workflows/rollback.yaml
Comment (Benbentwo) [reply .github/workflows/rollback.yaml:1]: Not sure why my git add added the file but not the content

**Review by milldr — APPROVED** [review]

**Pull request merged**